cmake/modules/gRPC.cmake

```diff
@@ -121,6 +128,7 @@ else()
     BUILD_BYPRODUCTS ${GRPC_LIB} ${GRPCPP_LIB}
     INSTALL_COMMAND ""
     CONFIGURE_COMMAND ""
+    UPDATE_DISCONNECTED TRUE
```

Comment:
Can we make the value a cache variable rather than hard-coding it?

This way we would give control to developers over whether or not to perform updates.

Another approach to achieve the same goal would be to define `EP_UPDATE_DISCONNECTED` directory options